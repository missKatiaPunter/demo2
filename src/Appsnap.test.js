import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

describe("test that App renders consistently", () => {
    test("matches a snapshot", () => {
        const tree = renderer.create(<App/>).toJSON();
        expect(tree).toMatchSnapshot();
    })
})