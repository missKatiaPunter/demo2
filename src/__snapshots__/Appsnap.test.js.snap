// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test that App renders consistently matches a snapshot 1`] = `
<div
  className="App"
>
  <div>
    <p>
      Hello
    </p>
    <button
      data-testid="counter-button"
      onClick={[Function]}
    >
      0
    </button>
  </div>
</div>
`;
